src/controllers/StellarContractController.ts

```
@@ -0,0 +1,218 @@
+import { StellarContractService } from "../services/StellarContractService";
+import {
+  MerchantRegistrationDTO,
+  TokenSupportDTO,
+  PaymentProcessingDTO,
+} from "../dtos/StellarContractDTO";
+import { AppError } from "../utils/AppError";
+import logger from "../utils/logger";
+import rateLimit, { RateLimitRequestHandler } from "express-rate-limit";
+
+// Helper function to format validation errors
+const formatValidationErrors = (errors: any[]) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)